core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -236,17 +236,15 @@ public boolean delete(AlluxioURI uri) {
    */
   public String getMountPoint(AlluxioURI uri) throws InvalidPathException {
     String path = uri.getPath();
-    String mountPoint = null;
 
     try (LockResource r = new LockResource(mReadLock)) {
       for (Map.Entry<String, MountInfo> entry : mMountTable.entrySet()) {
         String alluxioPath = entry.getKey();
-        if (PathUtils.hasPrefix(path, alluxioPath)
-            && (mountPoint == null || PathUtils.hasPrefix(alluxioPath, mountPoint))) {
-          mountPoint = alluxioPath;
+        if (!alluxioPath.equals(ROOT) && PathUtils.hasPrefix(path, alluxioPath)) {
```

Comment:
if `alluxioPath.equals(ROOT)`, why not return `ROOT` here?

Reply:
for example, if `/mnt` is a mount point and is a prefix for `path`, we would like to return `/mnt` rather than `/`.

Reply:
but in this case alluxioPath is `/` but not `/mnt`?

Reply:
the iteration order of hashmap is undefined, so maybe `/` is encountered first, but we actually want to return `/mnt` instead of `/`

Reply:
I see. That makes sense. `/` is a special path.